src/WebJobs.Extensions.OpenAI.Kusto/KustoSearchProvider.cs

```diff
@@ -171,15 +171,27 @@ KustoConnectionStringBuilder GetKustoConnectionString(string connectionName)
 
     static string GetEmbeddingsString(ReadOnlyMemory<float> embedding)
     {
-        ReadOnlySpan<float> span = embedding.Span;
-        StringBuilder builder = new(span.Length * 10); // 9 is the longest string length of a float + 1 for comma
-
-        for (int i = 0; i < span.Length; i++)
+        StringBuilder sb = new();
+        foreach (float value in embedding.Span)
         {
-            builder.Append(span[i]).Append(',');
+            sb.Append(value.ToString());
+            sb.Append(",");
         }
-        builder.Remove(builder.Length - 1, 1);
+        sb.Length--; // remove the trailing comma
+        return sb.ToString();
+    }
 
-        return builder.ToString();
+    static string GetEmbeddingsArray(ReadOnlyMemory<float> embedding)
+    {
+        StringBuilder sb = new();
+        sb.Append("[");
+        foreach (float value in embedding.Span)
+        {
+            sb.Append(value.ToString());
```

Comment:
Same question about using `value` instead of `value.ToString()`.

Reply:
updated to use value directly, let me know if it is as expected.